LEAF_Request_Portal/sources/Form.php

```diff
@@ -2605,16 +2606,14 @@ public function getCustomData(array $recordID_list, string|null $indicatorID_lis
                                 }
                             }
                             break;
-                        case 'orgchart_employee':
-                            $empRes = $this->employee->lookupEmpUID($item['data']);
-                            if (isset($empRes[0]))
-                            {
-                                $item['data'] = "{$empRes[0]['firstName']} {$empRes[0]['lastName']}";
-                                $item['dataOrgchart'] = $empRes[0];
-                            }
-                            else
-                            {
-                                $item['data'] = '';
+                        case 'orgchart_employee': //report builder cells
+                            $item['data'] = '';
+                            if (isset($item['metadata'])) {
```

Comment:
What happens if not "isset"?  Do we want a silent fail here?  Or do we want to catch it and provide something back?

Reply:
Item data will remain empty.  This is what it had done previously, but since it also needs to be checked for a valid name it was simpler to set it at the top and update it if it had a valid name rather than set it afterwards.